frontend/pages/shelter/pet/[id].tsx

```diff
@@ -0,0 +1,345 @@
+import { EditOutlined } from "@ant-design/icons";
+import {
+	Breadcrumb,
+	Button, Form, Space,
+	Table,
+	Tabs,
+	Tag
+} from "antd";
+import { Content } from "antd/lib/layout/layout";
+import Title from "antd/lib/typography/Title";
+import dayjs from "dayjs";
+import ShelterLayout from "layouts/shelter/ShelterLayout";
+import { useRouter } from "next/dist/client/router";
+import Image from "next/image";
```

Comment:
@stonefruit @chunqi @ShaunLWM 
[reference](https://github.com/bettersg/PawScore/commit/f65c0404d4c5d89d26cca37d56b93d30dc2f01f2)

Reply:
@chunqi May I know why we do not want the auto optimization feature? Just curious!